src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/VsProjectJsonNuGetProject.cs

```diff
@@ -72,17 +75,23 @@ protected override async Task UpdateInternalTargetFrameworkAsync()
                 var jsonTargetFramework = targetFramework as NuGetFramework;
                 if (IsUAPFramework(jsonTargetFramework))
                 {
+#pragma warning disable CS0618 // Type or member is obsolete
+                    // Need to validate no project systems get this property via DTE, and if so, switch to GetPropertyValue
```

Comment:
We have some telemetry on when the fallback is being used correct?
Does that telemetry contain the property? 

IOW, are we attempting to validate this?

Reply:
Just a question of time and prioritization. I created a follow up issue: https://github.com/NuGet/Client.Engineering/issues/2670